### ISSUE TITLE
[3.6] Print speed calculation error [Solved]

### ISSUE THREAD

**Issue by johnkray** [Ultimaker/Cura#5180]
**Application Version**
3.6

**Platform**
Windows 10

**Printer**
Custom printer

**Steps to Reproduce**
Copy all my attached cura config files to the program files for Cura 3.6. Launch cura, add the Hydra Research Nautilus from the add printer menue. Then seletct "normal (.15)" profile for any material.

**Actual Results**
Infill speed, wall speed, and top bottom wall speed do not calculate properly. They do calculate based on print speed but not to the expected values. For all other quality profiles (.1 .2 .3) the speeds do calculate according to the formulas defined.

**Expected results**
The printing speeds should calculate according to the formulas. For my formulas defined in my printer configuration the easiest way to  check if everything is working properly is to set the print speed to a value < 25 which should make all other print speed values that would normally be calculated be = to the print speed.

**Additional Information**
I have tried disableing my custom print speed formulas and using the default cura print speed formulas and for the normal (.15) profile the speed still calculate incorrectly, making me think there is some issue will all the "noirmal (.15)" profiles I have generated. However, I cannot cannot notice any issues with them. 

My custom Cura config files are in a zip at the bottom. I realize that my config is pretty involved, but hopefully some one can help. Thank you in advance!

Expected result with print speed >25 (with "draft (.2)" quality profile selected)
![screenshot 44](https://user-images.githubusercontent.com/37052613/51223954-cb1ad480-18f8-11e9-97b3-713d415480aa.png)

Expected result with print speed <25 (with "draft (.2)" quality profile selected)
![screenshot 45](https://user-images.githubusercontent.com/37052613/51223974-e554b280-18f8-11e9-9bf6-9af1079b714c.png)

Unexpected result with print speed >25 (with "normal (.15)" quality profile selected)
![screenshot 46](https://user-images.githubusercontent.com/37052613/51223978-e7b70c80-18f8-11e9-83ba-9da7462013e1.png)

Unexpected result with print speed <25 (with "normal (.15)" quality profile selected)
![screenshot 48](https://user-images.githubusercontent.com/37052613/51224137-7461ca80-18f9-11e9-997f-c611d23255bc.png)

All my custom printer configuration files.
[resources.zip](https://github.com/Ultimaker/Cura/files/2762469/resources.zip)

**Comment by diegopradogesto** [comment]
As far as I could investigate on this, I see the problem that you are setting the values in your quality profiles (that is correct) but you are still using the default global profiles. In case of the 0.15mm layer height, it's using the `fast.inst.cfg` that defines a different speed for the infill. 

For avoiding confusion, I think it's better for you to define your own global qualities. I tested and that fixes your issue.

BTW, I found that you have a typo in your machine definition file, using `Ture` instead of `True` in two settings ;-)

Hope it helps!
